pkg/controllers/clusterresourceplacementeviction/controller.go

```diff
@@ -209,7 +209,7 @@ func (r *Reconciler) executeEviction(ctx context.Context, validationResult *evic
 	}
 
 	// handle special case for PickAll CRP.
-	if crp.Spec.Policy.PlacementType == placementv1beta1.PickAllPlacementType {
+	if crp.Spec.Policy == nil || crp.Spec.Policy.PlacementType == placementv1beta1.PickAllPlacementType {
```

Comment:
Why the default value here does not take effect: https://github.com/Azure/fleet/blob/main/apis/placement/v1beta1/clusterresourceplacement_types.go#L136?

Reply:
can we follow the pattern to call defaulter.SetDefaultsClusterResourcePlacement() so we don't need to handle those cases specially